### PR TITLE
TicTacToe Game Project

### PR DESCRIPTION
**Pull request by Mupa1**
## TicTacToe Game Project

In this project, we made all changes requested by the TSE. We:-
- Divided the project into 3modules
- Separated the game logic from the dom manipulation
- Removed the numbers from the cell

Kindly review our project

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hi Team. 👋

**One has to love the frontend. This is neat.👌🚀**

However, let's fix this:

- [ ] Oh my there is no package.json in your project. It is very important to have a package.json. How does a user start your project? Please include it.

- [ ] Please add `Getting started` section to your readme.

**### There are other comments made on your code please  check and fix them.**

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

HAPPY CODING👨‍💻

File: index.js [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+  playerTwoName,
+  alertMsg,
+  gameOver,
+  // eslint-disable-next-line import/extensions
+} from './dom.js';
+
+// eslint-disable-next-line func-names
+const domElements = (function () {
+  const inAlertMsg = (msg) => {
+    alertMsg.innerHTML = msg;
```
Comment (mikenath223) [inline index.js]: - [ ] This still counts as DOM manipulation. Please move this to dom.js, along with other similar code lines.

File: index.js [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+    if (gameBordModule.count > 8) {
+      return true;
+    }
+    return false;
+  };
+
+  const checkNextMove = () => {
+    if (winChecker()) {
+      domElements.inGameOver(`${gameBordModule.currentPlayerName} Wins!`);
+      gameOver.style.display = 'block';
```
Comment (mikenath223) [inline index.js]: - [ ] All styles should be in the CSS file and then the dom should be manipulated by togling a classlist on dom.js.

**Comment by Samitti** [comment]
Hi, we have made the changes requested from previous review. Kindly review our code. 
-  added the package.json file.
- separated the dom manipulation from the game logic. 
- added the getting started section on the README.

**Review by rOluochKe — APPROVED** [review]
### Well done, you got it right :clap: The project has been approved. Congratulations :boom: :100:
### Merge and submit the project. Good luck with the next one.
- Happy coding!

**Pull request merged**